docs/integration_fetching_data.md

```diff
@@ -113,13 +119,11 @@ class MyEntity(CoordinatorEntity, LightEntity):
         super().__init__(coordinator)
         self.idx = idx
 
-    @property
-    def is_on(self):
-        """Return entity state.
-
-        Example to show how we fetch data from coordinator.
-        """
-        self.coordinator.data[self.idx]["state"]
+    @callback
+    def _handle_coordinator_update(self) -> None:
```

Comment:
For coordinator entities it's often simpler to use state properties instead of implementing `_handle_coordinator_update` as we can see in this change.

I'd suggest to keep the property `is_on` but mention `_handle_coordinator_update` separately.